src/sentry/static/sentry/app/utils/discover/genericDiscoverQuery.tsx

```diff
@@ -221,4 +213,21 @@ class GenericDiscoverQuery<T, P> extends React.Component<Props<T, P>, State<T>>
   }
 }
 
+type RequestParameters = Partial<EventQuery & LocationQuery>;
+
+export async function doDiscoverQuery<T>(
+  api: Client,
+  url: string,
+  params: RequestParameters
+): Promise<[T, string | undefined, JQueryXHR | undefined]> {
+  return api.requestPromise(url, {
+    method: 'GET',
+    includeAllArgs: true,
+    query: {
+      // marking params as any so as to not cause typescript errors
+      ...(params as any),
```

Comment:
You could also add a second generic to avoid the any, but since it's outgoing, meh.